lib/Cake/Test/Case/Cache/Engine/ApcEngineTest.php

```diff
@@ -31,8 +31,12 @@ class ApcEngineTest extends CakeTestCase {
  * @var string
  */
 	protected $_apcExtension = 'apc';
+	/**
+	 * @var array|int[]|mixed
+	 */
+	private $_cacheDisable;
```

Comment:
This property is added in a few test classes because I was getting warnings about it not being present before I had fixed the error-handling. It doesn't actually need to be there for the tests to pass but it also doesn't hurt to clean this up I guess.